.github/workflows/build_rpms.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build RPMs
+
+on:
+  # push:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  build_rpm_build_container:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repository: ManageIQ/manageiq-rpm_build
+    - name: Set up registry credentials
+      if: ${{ github.event_name != 'pull_request' }}
+      run: |
+        echo "REGISTRY_USERNAME=${{ secrets.REGISTRY_USERNAME }}" >> $GITHUB_ENV
+        echo "REGISTRY_PASSWORD=${{ secrets.REGISTRY_PASSWORD }}" >> $GITHUB_ENV
+    - name: Run container build script
+      run: bin/build_container_image
+  build_rpms:
+    needs: build_rpm_build_container
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up registry credentials
+      run: |
+        echo "REGISTRY_USERNAME=${{ secrets.REGISTRY_USERNAME }}" >> $GITHUB_ENV
+        echo "REGISTRY_PASSWORD=${{ secrets.REGISTRY_PASSWORD }}" >> $GITHUB_ENV
+    - name: Build and push RPMs
+      run: |
+        echo $REGISTRY_PASSWORD | docker login docker.io --password-stdin --username $REGISTRY_USERNAME
+        mkdir options
+        echo "---" > options/options.yml
+        echo "rpm_repository:" >> options/options.yml
+        echo "  digitalocean_access_token: ${{ secrets.RPM_BUILD_DO_ACCESS_TOKEN }}" >> options/options.yml
+        echo "  s3_api:" >> options/options.yml
+        echo "    access_key: ${{ secrets.RPM_BUILD_S3_ACCESS_KEY " >> options/options.yml
+        echo "    secret_key: ${{ secrets.RPM_BUILD_S3_SECRET_KEY " >> options/options.yml
```

Comment:
I think there's a slightly cleaner way to do this, but this is fine for now.  I do worry that it will expose the secrets like if the options.yml is printed or something.

Reply:
Yeah, we either have to write a file somewhere or modify the code to accept ENV variables or something.

Reply:
I'd prefer env vars moving forward - never really liked secrets in the options.